src/components/Explore/ExploreChart.tsx

```diff
@@ -17,6 +17,8 @@ const ExploreChart: React.FC<{
   barOpacity?: number;
   barOpacityHover?: number;
   isMobileXs?: boolean;
+  dateRange: Date[];
+  dataMeasure: DataMeasure;
```

Comment:
Also here, it might be better to pass this as a prop to format the values rather than an enum (the same formatter can be used for the tooltip and the y-axis)